server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSSendOptionsCheckUtility.java

```diff
@@ -246,6 +254,9 @@ private void checkText(CICSParser.Cics_send_textContext ctx) {
         "JUSTIFY or JUSFIRST or JUSLAST", ctx.JUSTIFY(), ctx.JUSFIRST(), ctx.JUSLAST());
     checkHasMutuallyExclusiveOptions(
         "HONEOM or L40 or L64 or L80", ctx.HONEOM(), ctx.L40(), ctx.L64(), ctx.L80());
+    if (noLengthOptionsEnabled() && !ctx.FROM().isEmpty()) {
+      checkHasMandatoryOptions(ctx.LENGTH(), ctx, "LENGTH");
+    }
```

Comment:
```suggestion
    if (noLengthOptionsEnabled()) {
      checkHasMandatoryOptions(ctx.LENGTH(), ctx, "LENGTH");
    }
```

The following code above can be removed as well because `FROM` is already a mandatory oprerand
```
    if (!ctx.LENGTH().isEmpty()) {
      checkHasMandatoryOptions(ctx.FROM(), ctx, "FROM");
    }
```